content/en/user-guide/aws/pipes/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "EventBridge Pipes"
+linkTitle: "EventBridge Pipes"
+description: Get started with EventBridge Pipes on LocalStack
+---
+
+## Introduction
+
+EventBridge Pipes allows users to create point-to-point integrations between event producers and consumers with transform, filter and enrich steps. Pipes is particularly useful for scenarios involving real-time data processing, application integration, and automated workflows, while simplifying the process of routing events between AWS services.
+
+LocalStack supports EventBridge Pipes via the Pro/Team offering, allowing you to use the Pipes APIs in your local environment to to create Pipes with SQS queues and Kinesis streams as source and target. You can also filter events using EventBridge event patterns and enrich events using Lambda.
+
+The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_pipes/), which provides information on the extent of MWAA's integration with LocalStack.
+
+## Getting started
+
+This guide is designed for users new to EventBridge Piples and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
```

Comment:
Unfortunately, our previews are currently down: https://www.saashub.com/surge-sh-status
so, I rendered it locally.

I tested the getting started guide and everything works 👏